Infrastructure/RestaurantReservation.Persistence/Profiles/MappingProfile.cs

```diff
@@ -0,0 +1,48 @@
+﻿using AutoMapper;
+using RestaurantReservation.Application.Entities;
+using RestaurantReservation.Application.DTOs;
+
+namespace RestaurantReservation.Persistence.Profiles
+{
+    public class MappingProfile : Profile
+    {
+        //TODO => Split to many profiles and use ReverseMap()
```

Comment:
Note: the IDEs recognise these TODOs and you  can find them in the TODO/Tasks tool.